src/sections/Community/Handbook/Handbook.style.js

```diff
@@ -1,585 +1,596 @@
-      }
-      ol > li > span {
-        color: ${(props) => props.theme.text};
-        transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-      }
+  .conduct-section {
+    @media screen and (min-width: 751px) {
+      margin-top: -90rem;
     }
+    @media screen and (min-width: 768px) and (max-height: 1145px) {
+      margin-top: -90vh !important;
+    }
+  }
 
-    h2 h3{
-      margin: 0.5rem 0;
-      color: ${(props) => props.theme.tertiaryColor};
-      transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  .page-section .heading-top {
+    @media screen and (min-width: 751px) {
+      padding-top: 1rem;
     }
+  }
 
-    .page-section{
-      h2{
-        padding-top: 7rem;
-        margin-top: -7rem;
-      }
-      h3{
-        padding-top: 7rem;
-        margin-top: -7rem;
-      }
-      @media (min-width: 750px) {
-        margin-top: -36rem;
-        margin-left: 20rem;
-      }
-      display: flex;
+  /* Sidebar */
+  .sidebar {
+    margin: 0;
+    padding: 0;
+    width: 250px;
+    background-color: var(--secondaryLightColor);
+    position: absolute;
+    height: 150rem;
+    overflow: auto;
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  }
 
-      @media screen and (min-width: 768px) and (max-height: 1145px) {
-       margin-top : -62vh ;
+  .sidebar a {
+    display: block;
+    color: black;
+    padding: 16px;
+    text-decoration: none;
+  }
 
+  .sidebar a.active {
+    background-color: var(--primaryLightColorTwo);
+    color: white;
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
+  }
 
+  .sidebar a:hover:not(.active) {
+    background-color: var(--secondaryLightColor);
+    color: white;
+    transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
   }
-      @media screen and (min-width: 1280px) and (max-width: 1350px) {
-        margin-left:16rem;
-      }
-    }
-    .conduct-section{
-      @media screen and (min-width: 751px) {
-        margin-top:-43rem;
-      }
-            @media screen and (min-width: 768px) and (max-height: 1145px) {
-       margin-top : -66vh ! important;
 
+  /* Page Header */
+  .page-header-section {
+    height: 10rem;
+    text-align: center;
+    background: linear-gradient(
+      250deg,
+      rgba(71, 126, 150, 1) 0%,
+      rgba(0, 179, 159, 1) 35%,
+      rgba(60, 73, 79, 1) 100%
+    );
+    h1 {
+      line-height: 10rem;
+      color: white;
     }
+  }
 
+  /* Community Home */
+  .community-home-subtitle {
+    text-align: center;
+    padding-top: 3rem;
+    padding-bottom: 1.5rem;
   }
 
-    }
-    .page-section .heading-top{
-      @media screen and (min-width: 751px) {
-        padding-top:1rem;
-      }
-    }
-    .sidebar {
-        margin: 0;
-        padding: 0;
-        width: 250px;
-        background-color:${(props) => props.theme.secondaryLightColor};
-        position: absolute;
-        height: 150rem;
-        overflow: auto;
-        transition: 0.8s cubic-bezier(0.2, 0.8, 0.2, 1);
-    }
+  #sign-off {
```

Comment:
Hi, LGTM overall.
But i have doubt about selector with id, can we do it without?